_specs/ecip-1001.md

```diff
@@ -0,0 +1,86 @@
+---
+lang: en
+ecip: 1001
+title: ECIP Code of Conduct
+status: Draft
+type: Meta
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/238
+author: Yaz Khoury (@YazzyYaz)
+created: 2019-12-03
+license: CC-By-4.0 License
+---
+This Code of Conduct ECIP has been taken directly from [Contributor Covenant Open Source Code of Conduct](https://www.contributor-covenant.org/)
+under the [CC-By-4.0 License](https://github.com/ContributorCovenant/contributor_covenant/blob/release/LICENSE.md)
+
+This Code of Conduct ECIP applies to all members wishing to participate in the ECIP process:
+1) ECIP Editors
+2) ECIP Coordinators
+3) Hardfork Coordinators
+4) ECIP Authors
+5) Ethereum Classic Community Members Participating in the ECIP Process.
+
+
+## Our Pledge
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards
+Examples of behavior that contributes to a positive environment for our community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others’ private information, such as a physical or email address, or real names, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Enforcement Responsibilities
+Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
```

Comment:
the only case i'd argue seems acceptable to delete something on behalf of someone else is in the event of dox

Reply:
> and will communicate reasons for moderation decisions when appropriate

They should communicate all decisions regardless of whether they think it's appropriate or not, Giving "community leaders" the ability to enforce / decide what content is/isn't acceptable and then not holding them accountable for their actions is giving a select few an abnormally high level of power over the project.

Reply:
@BelfordZ agreed that doxing is the main reason for this doc.

We certainly want to allow for instigation and dissent in our community. But I worry there are things worse than doxing too that we might leave out.